.mvn/maven.config

```diff
@@ -0,0 +1 @@
+-ssettings.xml
```

Comment:
is used so that the settings.xml is used during the build. This is then the case locally as well as in the action on github